src/chunk.c

```diff
@@ -2178,6 +2189,14 @@ get_chunks_in_time_range(Hypertable *ht, int64 older_than, int64 newer_than, Mem
 	end_strategy = (older_than == PG_INT64_MAX) ? InvalidStrategy : BTLessStrategyNumber;
 	time_dim = hyperspace_get_open_dimension(ht->space, 0);
 
+	if (time_dim == NULL)
+		time_dim = hyperspace_get_closed_dimension(ht->space, 0);
+
+	Ensure(time_dim != NULL,
+		   "time dimension not found for hypertable \"%s\".\"%s\"",
```

Comment:
@fabriziomello should the message be: "partitioning dimesion not found for hypertable"?

Reply:
Fixed!